.github/workflows/docker-build-push-railgun.yaml

```
@@ -0,0 +1,79 @@
+name: Build and push Railgun images
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'next'
+    tags:
+      - '2.*'
```

```suggestion
      - '2.*'
      - 'v2.*'
```

same reasoning as https://github.com/Kong/kubernetes-ingress-controller/pull/1410/files#r649090835